### ISSUE TITLE
Missing Versions

### ISSUE THREAD

**Issue by Virusnest** [JJTech0130/meta-multimc#9]
This is outdated and needs updating it is missing the latest versions of fabric and vanilla  1.16.3 and 1.16.3 prerelease

**Comment by JJTech0130** [comment]
Yes. See README if you want to update it yourself, otherwise please wait for me to fix the build bot.

**Comment by JJTech0130** [comment]
0

**Comment by Virusnest** [comment]
Ok Thanks.

**Comment by JJTech0130** [comment]
Perhaps I will host an S3 instance for this in the near future-- should be easier for my script to update it.

**Comment by JJTech0130** [comment]
Preview S3 bucket: https://dl.jjtech.dev/meta/aarch64/v1/
When I get some time I will finish my auto-update script.

**Comment by nathanejohnson** [comment]
so I'm just now dipping my toes into multimc, so I'm not really sure what the scope of your build bot is.  However, I just banged together this script to try to update from upstream meta and wondered if this might be of some use:

https://github.com/nathanejohnson/meta-multimc/blob/master/copy_meta_multimc.sh

If so I could do open a PR.

**Comment by nathanejohnson** [comment]
on a related note, what's the likelihood of getting this effort (i.e. your fork) rolled into the upstream multimc?
